### PR TITLE
Add SQS

### PR DESCRIPTION
**Pull request by manabusakai**
This pull request to add the SQS.

I have been helped by Terraforming :smile:

### CODE REVIEW

**Comment by dtan4** [comment]
:eyes:

**Comment by manabusakai** [comment]
The rest is a specs.
Sorry..., it takes time to write a specs.

**Comment by manabusakai** [comment]
I removed [WIP] to PR title.

I was waiting for that fix of this bug ( https://github.com/aws/aws-sdk-ruby/issues/1099 ).

File: lib/terraforming/template/tf/sqs.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<% queues.each do |queue| -%>
+resource "aws_sqs_queue" "<%= module_name_of(queue) %>" {
+    name                       = "<%= module_name_of(queue) %>"
+    visibility_timeout_seconds = <%= queue["VisibilityTimeout"] %>
+    message_retention_seconds  = <%= queue["MessageRetentionPeriod"] %>
+    max_message_size           = <%= queue["MaximumMessageSize"] %>
+    delay_seconds              = <%= queue["DelaySeconds"] %>
+    receive_wait_time_seconds  = <%= queue["ReceiveMessageWaitTimeSeconds"] %>
+}
```
Comment (suthio) [inline lib/terraforming/template/tf/sqs.erb]: sqs Not included  `polixy` 
https://www.terraform.io/docs/providers/aws/r/sqs_queue.html#policy

Comment (suthio) [inline lib/terraforming/template/tf/sqs.erb]: @manabusakai 
added policy
https://github.com/manabusakai/terraforming/pull/1

**Comment by manabusakai** [comment]
Add two argument, `policy` and `redrive_policy`.
Special thanks to @suthio :+1:

**Comment by dtan4** [comment]
It works fine. LGTM :+1: Thanks :exclamation: 
Sorry for late review :bow:

**Pull request merged**